site/src/layouts/DetailComponent/DetailComponent.module.css

```diff
@@ -28,3 +28,7 @@
 .overlay {
   max-width: 100vw;
 }
+
+.content {
```

Comment:
is it just an issue with this layout ?

Reply:
is this fixing tablet layout? but it makes desktop content alignment to the left (not aligned to heading section)